phpunit.xml.dist

```diff
@@ -4,7 +4,12 @@
          colors="true"
          beStrictAboutOutputDuringTests="true"
          displayDetailsOnPhpunitDeprecations="true"
-         displayDetailsOnTestsThatTriggerDeprecations="true"
+         displayDetailsOnTestsThatTriggerErrors="true"
```

Comment:
Regarding `displayDetailsOnTestsThatTriggerErrors`: what's the difference between having it and the default behavior?